tests/console/sshd.pm

```diff
@@ -94,45 +98,61 @@ sub run {
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 22 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 4242 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-    assert_script_run "ssh -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 5252 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
-
-    assert_script_run "ssh -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
-    assert_script_run "ssh -t -o ProxyCommand='ssh $ssh_testman\@localhost nc localhost 4242' $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 5252 $ssh_testman\@localhost whoami";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 22 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 4242 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+    assert_script_run "ssh -v -p 4242 $ssh_testman\@localhost 'ssh-keyscan -p 5252 localhost 127.0.0.1 ::1 >> ~/.ssh/known_hosts'";
+
+    assert_script_run "echo 'sshd.pm: testing ssh proxy' | logger";
+    assert_script_run "ssh -v -p 4242 -tt $ssh_testman\@localhost ssh -tt $ssh_testman\@localhost whoami";
+    sleep 60;
+    assert_script_run "ssh -v -t -o ProxyCommand='ssh -v $ssh_testman\@localhost nc localhost 4242' $ssh_testman\@localhost whoami";
+    sleep 60;
     if (is_leap('15.0+') || is_tumbleweed || is_sle('15+')) {
-        assert_script_run("ssh -J $ssh_testman\@localhost:4242 $ssh_testman\@localhost whoami");
+        assert_script_run("ssh -v -J $ssh_testman\@localhost:4242 $ssh_testman\@localhost whoami");
     }
 
     # SCP (poo#46937)
+    assert_script_run "echo 'sshd.pm: testing scp' | logger";
     assert_script_run "scp -4v $ssh_testman\@localhost:/etc/resolv.conf /tmp";
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/{group,passwd}' /tmp";
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/ssh/*.pub' /tmp";
 
     # Restore ~/.ssh generated in consotest_setup
     # poo#68200. Confirm the ~/.ssh_bck directory is exist in advance and then restore, in order to avoid the null restore
+    assert_script_run "echo 'sshd.pm: removing ~/.ssh' | logger";
     assert_script_run 'rm -rf ~/.ssh';
     assert_script_run 'if [ -d ~/.ssh_bck ]; then mv ~/.ssh_bck ~/.ssh; fi';
 
+    assert_script_run "echo 'sshd.pm: killing $ssh_testman user processes' | logger";
     assert_script_run "killall -u $ssh_testman || true";
     wait_still_screen 3;
+
+    assert_script_run "journalctl -b > /tmp/journalctl-b.log";
+    upload_logs "/tmp/journalctl-b.log";
+    my $journalctlgrep = "grep -A6 -B24 'segfault\\|fatal' /tmp/journalctl-b.log";
+    if (script_run($journalctlgrep) == 0) {
+        record_info("journalctl", "Please check the journalctl!\n" . script_output($journalctlgrep));
+    }
```

Comment:
i wonder if this should be in the `post_run_hook`

Reply:
I would consider to move this to the base `consoletest` but let's do it separately.